### PR TITLE
chore(weave): Initial Compare View

### PR DESCRIPTION
**Pull request by tssweeney**
This PR unlocks evaluation comparison by allowing users compare rows the evaluation pivot table. Effectively this allows them to compare line-by-line metrics for each example for a given dataset.

Screenshot 2024-01-29 at 17 30 00
Screenshot 2024-01-29 at 17 29 54

### CODE REVIEW

File: weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CompareCallsPage.tsx [hunk 1]
```diff
@@ -0,0 +1,474 @@
...
+import {WFCall} from './wfInterface/types';
+
+export const CompareCallsPage: React.FC<{
+  entity: string;
+  project: string;
+  callIds?: string[];
+  primaryDim?: string;
+  secondaryDim?: string;
+}> = props => {
+  const [selectedOpVersion, setSelectedOpVersion] = React.useState<
```
Comment (jamie-rasmussen) [inline weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CompareCallsPage.tsx:27]: React.useState => useState

File: weave-js/src/components/PagePanelComponents/Home/Browse3/pages/util.tsx [hunk 2]
```diff
@@ -32,6 +33,17 @@ export const renderCell = (value: any) => {
...
     return value ? 'True' : 'False';
   }
+  if (typeof value === 'number') {
+    return (
+      <Box
+        sx={{
+          textAlign: 'right',
+          width: '100%',
+        }}>
+        {value.toFixed(4)}
```
Comment (jamie-rasmussen) [inline weave-js/src/components/PagePanelComponents/Home/Browse3/pages/util.tsx:43]: I think we want to skip this if the value is an integer, like `true_count`?

**Review by jamie-rasmussen — APPROVED** [review]

File: weave-js/src/components/PagePanelComponents/Home/Browse3/pages/util.tsx (hunk 2 above)
Comment (tssweeney) [reply weave-js/src/components/PagePanelComponents/Home/Browse3/pages/util.tsx:43]: i like it - will make this change

File: weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CompareCallsPage.tsx (hunk 1 above)
Comment (tssweeney) [reply weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CompareCallsPage.tsx:27]: Damn - this is all auto imported and done in my IDE. Need to figure out how to prefer the destructured input.

Comment (tssweeney) [reply weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CompareCallsPage.tsx:27]: Ok, changed this in all files i touched

**Pull request merged**